js/page-specific/phone-directory/phone-directory.js

```diff
@@ -0,0 +1,131 @@
+var jQuery = $.noConflict(true);
+var pageCount = 0;
+
+function __onPrevPage() {
+    for (var i = 1; i <= pageCount; i++) {
+        if (document.getElementById("Page" + i).style.display == "block") {
+            document.getElementById("Page" + i).style.display = "none";
+            document.getElementById("Page" + (i - 1)).style.display = "block";
+            break;
+        }
+    }
+    setNextPrevious(i - 1);
+}
+
+function setNextPrevious(pageNum) {
+    if (pageNum < pageCount) {
```

Comment:
you could really clean this up nicely by doing something like this.

```js
$('#Next').toggle(pageNum < pageCount);
```

see https://api.jquery.com/toggle/#toggle-display

could apply the same rules to the conditional starting on 22.

Reply:
Thanks, updated this